docs/source/concept_models/vivarium_sanofi_multiple_myeloma/concept_model.rst

```diff
@@ -36,34 +36,70 @@ Vivarium - CSU - Simulating Multiple Myeloma registries
 +------------------------------------+
 | List of abbreviations              |
 +=======+============================+
-|       |                            |
+|   MM  | Multiple myeloma           |
++-------+----------------------------+
+|   RR  | Relapsed/ Refractory MM    |
++-------+----------------------------+
+|   isa.| Isatuximab                 |
 +-------+----------------------------+
 
+
 .. _1.0:
 
 1.0 Background
 ++++++++++++++
+For this project, we are interested in the expected impact of Isatuximab on the overall Unites States population, among a registry population, and in key-subpopulations listed below, in comparison to a business as usual treatment scenario over 5 years.
+
+Key sub-populations agreed upon include: 
+
+* Black/African American population 
+
+* Population with COPD 
+
+* Population with renal impairment 
 
+In the absence of data on treatment effects for sub-groups we will need to make assumptions with client guidance. A strength of this project is that we will be able to iteratively update such assumptions as we get real world data from registries and re-run the simulation. 
 
 .. _1.1:
 
 1.1 Project overview
 --------------------
-
+This project intends to model the impact of a new treatment option for MM, Isatuximab, from 2021 to 2026 among the general population, key subpopulations listed above, and a registry population within the United States. The model will make use of the current multiple myeloma treatment guidelines in the United States, which are dependent on cancer stage, age, comorbidities, eligibility for stem cell transplantation, and risk category.  
```

Comment:
Isa. is a new type of treatment for RRMM instead of MM.

Reply:
Thank you for the correction! I have made this update.